docker-compose.yml

```diff
@@ -262,6 +262,28 @@ services:
       - otelcol
     logging: *logging
 
+  quoteservice:
+    image: ${IMAGE_NAME}:${IMAGE_VERSION}-quoteservice
+    container_name: quoteservice
+    build:
+      context: ./
+      dockerfile: ./src/quoteservice/Dockerfile
+    ports:
+      # remove localhost port b4 merging
+      - "${QUOTE_SERVICE_PORT}:${QUOTE_SERVICE_PORT}"
+    environment:
+      # OTEL_EXPORTER_OTLP_TRACES_ENDPOINT # Not working for PHP
+      - QUOTE_SERVICE_PORT
+      - OTEL_SERVICE_NAME=quoteservice
+      - OTEL_EXPORTER_OTLP_ENDPOINT=otelcol:4317
```

Comment:
I'm not using the `.env` `OTEL_EXPORTER_OTLP_ENDPOINT`, because it has a `http://` in front of it, which doesn't work with PHP.